Tests/test_file_jpeg.py

```diff
@@ -255,6 +255,21 @@ def test_exif_gps_typeerror(self):
         # Should not raise a TypeError
         im._getexif()
 
+    def test_interop(self):
+        # Arrange
+        im = Image.open('Tests/images/flower.jpg')
+
+        # Act
+        exif = im._getexif()
+
+        # Assert
+        self.assertEqual(exif["interop"], {
+            1: 'R98',
+            2: b'0100',
+            4097: 2272,
+            4098: 1704
```

Comment:
Tip: including a comma means when the next one is added, the diff is smaller:
```suggestion
            4098: 1704,
```